test/cpp/qps/driver.cc

```diff
@@ -650,6 +660,11 @@ std::unique_ptr<ScenarioResult> RunScenario(
     rrc->set_status_code(it->first);
     rrc->set_count(it->second);
   }
+
+  // Fill in start and end time for the test scenario
+  result->mutable_summary()->mutable_start_time()->CopyFrom(start_timestamp);
```

Comment:
Here, if you use start_time and end_time of type time_t, you can just call set_seconds(start_time) on the mutable_start_time() to set the time. Since we are using proto3, it is not necessary to call set_nanos(0).